modules/mobile/spec/request/appointments_create_request_spec.rb

```diff
@@ -25,35 +25,48 @@
     allow_any_instance_of(Mobile::V0::VAOSAppointments::AppointmentsHelper).to receive(:get_clinic).and_return(mock_clinic)
     allow_any_instance_of(Mobile::V0::VAOSAppointments::AppointmentsHelper).to receive(:get_facility).and_return(mock_facility)
     # rubocop:enable Layout/LineLength
+
+    @original_cassette_dir = VCR.configure(&:cassette_library_dir)
+    VCR.configure { |c| c.cassette_library_dir = 'modules/mobile/spec/support/vcr_cassettes' }
```

Comment:
I copied all the VCR cassettes over from VAOS because that's what I've previously been told we should do.